website_product_document_download_counter/models/product_document.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2025 Cetmix OÜ
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/LGPL).
+
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+
+
+class ProductDocument(models.Model):
+    _inherit = "product.document"
+
+    download_count = fields.Integer(
+        default=0,
+        readonly=True,
+        help="Number of times this document has been downloaded",
+    )
+
+    count_downloads = fields.Boolean(
+        default=False,
```

Comment:
`False` is the default value for Boolean fields, no need to put this value explicitly.
Also this field should not be copied too.